src/Orc.SystemInfo/Wmi/WindowsManagementConnection.cs

```diff
@@ -6,15 +6,14 @@
     using Catel.Logging;
     using Orc.SystemInfo.Win32;
 
-    public sealed class WindowsManagementConnection : IDisposable
+    public sealed class WindowsManagementConnection : Catel.Disposable
```

Comment:
Catel.-> reductant namespace...it's already there in using directives

Reply:
Good point i think i will do a bit cleanup for whole project, may be there is more issue like these